scrutiny.json

```
@@ -0,0 +1,82 @@
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for Scrutiny config files. Eg: create a Share called Scrutiny-Share-Config for this purpose alone.",
+                        "label": "Config share"
+                    }
+                },
+                "environment": {
+                    "PUID": {
+                        "description": "Enter a valid system UserID for Scrutiny. Must have full permissions on the previous Share.",
+                        "label": "UID",
+                        "index": 1
+                    },
+                    "PGID": {
+                        "description": "Enter a valid system GroupID for Scrutiny. This group or the above UserID must have full permissions on the previous Share.",
+                        "label": "GID",
+                        "index": 2
+                    },
+                    "SCRUTINY_API_ENDPOINT": {
+                        "description": "API endpoint of the scrutiny UI. Do not change unless using as a remote collector. Default: 'http://localhost:8080'",
+                        "label": "SCRUTINY_API_ENDPOINT",
+                        "index": 3
+                    },
+                    "SCRUTINY_WEB": {
+                        "description": "Run the web service. Default: 'true'",
+                        "label": "SCRUTINY_WEB",
+                        "index": 4
+                    },
+                    "SCRUTINY_COLLECTOR": {
+                        "description": "Run the metrics collector. Default: 'true'",
+                        "label": "SCRUTINY_COLLECTOR",
+                        "index": 5
+                    }
+                }
+            }
+        },
+        "description": "<p>WebUI for smartd S.M.A.R.T monitoring. Scrutiny is a Hard Drive Health Dashboard &amp; Monitoring solution, merging manufacturer provided S.M.A.R.T metrics with real-world failure rates from Backblaze.</p><p><strong>Note: </strong>In order to access the devices, this container will run with privileged capabilities. More info can be found <a href='https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities' target='_blank'>here</a>.</p><p>Based on custom docker image: <a target='_blank' href='https://hub.docker.com/r/linuxserver/scrutiny'>https://hub.docker.com/r/linuxserver/scrutiny</a>, available for x86-64, arm64, and armhf</p>",
+		"more_info": "<p><strong>Note:</strong>This image is updating the stats only once a day, if you see: '<i>No Devices Detected!</i>' after the installation, just wait 24 hours. <br>This process can also be triggered manually from the shell by executing the following command: <code>docker exec scrutiny scrutiny-collector-metrics run</code><br>More documentation on this can be found here: <a href='https://github.com/AnalogJ/scrutiny#usage'>Scrutiny GitHub</a></p>",
```

Open the Github link in a new page:
```suggestion
		"more_info": "<p><strong>Note:</strong>This image is updating the stats only once a day, if you see: '<i>No Devices Detected!</i>' after the installation, just wait 24 hours. <br>This process can also be triggered manually from the shell by executing the following command: <code>docker exec scrutiny scrutiny-collector-metrics run</code><br>More documentation on this can be found here: <a href='https://github.com/AnalogJ/scrutiny#usage' target='_blank'>Scrutiny GitHub</a></p>",
```